tfhe/src/core_crypto/gpu/slice.rs

```diff
@@ -93,20 +93,24 @@ where
     ///
     /// - [CudaStreams::synchronize] __must__ be called after the copy as soon as synchronization is
     ///   required.
-    pub unsafe fn copy_from_gpu_async(&mut self, src: &Self, streams: &CudaStreams, gpu_index: u32)
-    where
+    pub unsafe fn copy_from_gpu_async(
+        &mut self,
+        src: &Self,
+        streams: &CudaStreams,
+        stream_index: u32,
```

Comment:
Ok, I guess it makes sense to rename this variable since the index is on the stream array and not on the gpu array.